Source/Extensions/Blazorise.LoadingIndicator/LoadingIndicatorService.cs

```diff
@@ -0,0 +1,192 @@
+﻿#region Using directives
+using System;
+using System.Collections.Generic;
+using System.Linq;
+#endregion
+
+namespace Blazorise.LoadingIndicator
+{
+    /// <summary>
+    /// A service to control LoadingIndicator components
+    /// </summary>
+    public class LoadingIndicatorService
```

Comment:
I think it would be "cleaner" to have an interface `ILoadingIndicatorService`. And then the interface would be used by the DI.